python/ray/tune/search/skopt/skopt_search.py

```diff
@@ -75,6 +75,8 @@ class SkOptSearch(Searcher):
         convert_to_python: SkOpt outputs numpy primitives (e.g.
             ``np.int64``) instead of Python types. If this setting is set
             to ``True``, the values will be converted to Python primitives.
+        optimizer_kwargs: Parameters to pass to the SkOpt optimizer.
+            (See ``skopt.optimizer.Optimizer`` for details of Parameters)
```

Comment:
Can we update this to say that this will only be used if the initialized `optimizer` parameter is not set?

Reply:
Thanks for your suggestion and modified.